scenes/start.ts

```diff
@@ -27,33 +27,68 @@ scene.step(async (ctx) => {
   ctx.session.message = message
   ctx.session.buttons = buttons
   ctx.session.regions = keyboardMessage
+  ctx.session.currPage = 1
+  ctx.session.keyboard = keyboard
 
   await ctx.reply(message, { reply_markup: buttons })
 })
 
 // fasting
 scene.wait('fasting').on('callback_query:data', async (ctx) => {
-  if (!ctx.session.regionId.includes(+ctx.update.callback_query.data)) {
-    return ctx.answerCallbackQuery(HLanguage('wrongSelection'))
-  }
+  const inputData = ctx.update.callback_query.data
 
-  await ctx.answerCallbackQuery()
+  if (ctx.session.regionId.includes(+inputData) || ['<', '>'].includes(inputData)) {
```

Comment:
How does `regionId` can be an array?

Reply:
copy/paste issue

Reply:
wuni anu usuldan qib fix qvorolismi pls. man endi 17 regiondaman teysam yana 0 dan ketadi

Reply:
what method?)

Reply:
hold your horses. Man i think its not a bug, (feature)

Reply:
![image](https://github.com/Xayrulloh/ReminderBot/assets/91729108/e04d7fa7-0262-4ec8-84e1-a4c973b6fa89)
as u can see it's possible

Reply:
I recommend you to rereview

Reply:
why?